spec/dummy/bin/dev

```diff
@@ -8,4 +8,19 @@ fi
 # Default to port 3000 if not specified
 export PORT="${PORT:-3000}"
 
-exec foreman start -f Procfile.dev --env /dev/null "$@"
+# If REDIS_ENABLED is true, create a temporary Procfile with Sidekiq
+if [ "$REDIS_ENABLED" = "true" ]; then
```

Comment:
this and the changes to the next couple files make it a little easier to run w/ redis and sidekiq on any app. just run REDIS_ENABLED=true bin/dev, this will enable caching with redis and run a sidekiq worker alongside the main server.